app/views/appropriate_bodies/participants/index.html.erb

```diff
@@ -10,17 +12,6 @@
   title: "Search",
   hint: "For example, school, participant name or teacher reference number (TRN)",
```

Comment:
could you update the content here to match the prototype?

Reply:
Updated